lib/inbox.dart

```diff
@@ -24,41 +23,181 @@ class Inbox extends StatelessWidget
         return email;
     }
 
-    String displayString( int docIndex, 
-                                    List<QueryDocumentSnapshot<Object?>> data )
+    Widget showRubricDetail(BuildContext context, doc)
     {
-        return 'Your resource ${data[ docIndex ][ 'submittedName' ]} has been ${data[ docIndex ][ 'status' ]}. \n' +
-               'Description: \n ${data[ docIndex ][ 'description' ]}\n' + 
-               'Additional Information: ${data[ docIndex ][ 'comments' ]}\n'
-               'Timestamp: ${data[ docIndex ][ 'timestamp' ]}';
+        String parts = doc['description'].replaceAll(', ', '\n');
+
+        // Avoid bad state if key DNE
+        String reviewer;
+        try
+        {
+            reviewer = doc['reviewedby'];
+        }catch(error)
+        {
+            reviewer = "";
+        }
+
+        return SimpleDialog(
+            title: Center(
+                child: Text(
+                    'Rubric Information',
+                    style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold),
+                ),
+            ),
+            contentPadding: EdgeInsets.all(16),
+            children: [
+                FractionallySizedBox(
```

Comment:
Is FractionallySizedBox necessary here? It seems like it's not doing anything, and we might replace it by moving up the child Column.